### PR TITLE
[Display_01] Define DisplayGpsReceiverImplementationTests

### PR DESCRIPTION
**Pull request by EC01EC**
Created test methods for DisplayGpsReceiverImplementationTests.puml

Fixes for issues #56 and #74

### CODE REVIEW

File: docs/uml-class-diagrams/display01/test/DisplayGpsReceiverImplementationTests/DisplayGpsReceiverImplementationTests.puml [hunk 1]
```diff
@@ -2,7 +2,9 @@
 
 package edu.oakland.test.display01 {
     class DisplayGpsReceiverImplementationTests {
-
+        + void measureGpsSignalStrength()
+        + void ifSignalStrengthGoodMeasureSignal()
+        + void elseSignalStrengthBadRejectSignal()
```
Comment (andrewdimmer) [inline docs/uml-class-diagrams/display01/test/DisplayGpsReceiverImplementationTests/DisplayGpsReceiverImplementationTests.puml]: Make sure you have a check that the next link in the chain is not missing!
```suggestion
        + void elseSignalStrengthBadRejectSignal()
        + void displayGpsManagerIsNotNull()
```

**Review by andrewdimmer — CHANGES_REQUESTED** [review]
One minor change, then you're set to merge!

**Review by andrewdimmer — APPROVED** [review]
Looks good to me, you're all set to merge!

**Review by stevecatherman — APPROVED** [review]
:)

**Pull request merged**